### PR TITLE
adding readme

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Adding updated README file containing the content we feel we need.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,168 @@
...
+
+1. Once the above is done, cd into the content repo.
+
+1. Run the command `yarn install` to fetch the latest packages and get the local
+   MDN testing environment set up. It is also recommended that you run
+   `yarn install` before every update you do to the source, to make sure you
+   have the latest packages.
+
+1. Next, create an environment variable called `CONTENT_TRANSLATED_ROOT`, which
+   points to the path to the translated-  content repo’s `files` directory. You
```
Comment (fiji-flo) [inline README.md:67]:
```suggestion
   points to the path to the *translated-content* repo’s `files` directory. You
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,168 @@
...
+
+### Working in the translated-content repo
+
+Over in the translated-content repo, decide what change you want to make, and
+then:
+
+1. Create a new branch to make your changes in.
+
+1. Switch to your new branch and make the changes you want to make. You can
+   keep going back to `localhost:5000` to test your changes and make sure the
```
Comment (fiji-flo) [inline README.md:92]: `localhost:5000/<your_locale>` (e.g. `localhost:5000/fr` for French)

**Review by fiji-flo — CHANGES_REQUESTED** [review]
All good just the two comments.

**Comment by chrisdavidmills** [comment]
@fiji-flo great, thanks for the review! I've fixed those issues, and will now get it merged.

**Pull request merged**